tests/ccnode_test.cpp

```diff
@@ -0,0 +1,69 @@
+#include "simulator/ccnode.h"
+#include "gtest/gtest.h"
+
+TEST(CCNodeTest, Constructor) {
+  et_simulator::CCNode * node = new et_simulator::CCNode(1, 2, 3, 0);
+  ASSERT_EQ(node->getMethodId(), 1);
```

Comment:
Thanks for the extensive tests! But, instead of ASSERT_EQ, EXPECT_EQ should be used here. ASSERT should be used in cases where the test cannot continue without running into an error.  For instance, I would ASSERT something is not a nullptr before dereferencing it later in the test but I would use EXPECT when comparing the output of a function with the expected value.

Reply:
Replaced assert to expect